packages/snap/src/utils/params.ts

```diff
@@ -0,0 +1,29 @@
+import { SignTransactionsParams, NearNetwork } from "../interfaces";
+
+export function isValidSignTransactions(params: unknown): asserts params is {
+  signTransactions: SignTransactionsParams;
+} {
+  if (
+    !(
+      params != null &&
+      typeof params == "object" &&
+      "signTransactions" in params &&
+      // @ts-expect-error
```

Comment:
its legit error as you didn't check params.signTransactions is object.
@BeroBurny @irubido maybe consider joi to do request validation?

Reply:
or maybe some lighter lib that ain't gonna cause ses error

Reply:
foud new favorite `superstruct`